delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/serialization/EventSerializer.scala

```diff
@@ -0,0 +1,105 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.serialization
+
+import akka.actor.ExtendedActorSystem
+import akka.serialization.SerializerWithStringManifest
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.CompositeViews
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeView.RebuildStrategy
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model.CompositeViewSource.AccessToken
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.model._
+import ch.epfl.bluebrain.nexus.delta.plugins.compositeviews.serialization.EventSerializer.compositeViewsEventManifest
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.context.JsonLdContext.keywords
+import ch.epfl.bluebrain.nexus.delta.sdk.model.Event
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.instances._
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.semiauto.deriveConfiguredCodec
+import io.circe.parser.decode
+import io.circe.syntax._
+import io.circe.{Codec, Decoder, Encoder, Printer}
+
+import java.nio.charset.StandardCharsets
+import scala.annotation.nowarn
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+/**
+  * A json serializer for Composite view plugin [[Event]] types.
+  */
+@nowarn("cat=unused")
+@SuppressWarnings(Array("UnusedMethodParameter"))
+class EventSerializer(system: ExtendedActorSystem) extends SerializerWithStringManifest {
+
+  private val printer: Printer = Printer.noSpaces.copy(dropNullValues = true)
+
+  override def identifier: Int = 453226
```

Comment:
this is the same identifier as elasticsearch. It will give problems when running it. Pick a different one.